app/assets/stylesheets/modules/footer.scss

```diff
@@ -0,0 +1,45 @@
+@import "variables";
+
+.footer {
+  background: $blue;
+  color: $white;
+  height: 300px;
+  position: absolute;
+  bottom: 0px;
+
+  .header {
+    font-family: Lato;
+    padding-top: 20px;
+    text-align: center;
+  }
+
+  h2 {
+    font-family: Lato;
+    line-height: 1.4;
+    color: $white;
+  }
+
+  hr {
+    border: solid #dddddd;
+    border-width: 1px 0 0;
+    clear: both;
+    height: 0;
+    margin: 1.25rem 0 1.1875rem;
+    margin-left: 650px;
+    margin-right: 650px;
+  }
+  .gen {
+    -moz-border-radius: 50%;
+    -ms-border-radius: 50%;
+    -o-border-radius: 50%;
+    -webkit-border-radius: 50%;
+    position: relative;
+    width: 100px;
+    overflow: hidden;
+    height: 100px;
+    border-radius: 50%;
+    img {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3